### ISSUE TITLE
ibm-radio-group AVT issue

### ISSUE THREAD

**Issue by aaratichavan** [IBM/carbon-components-angular#1642]
ibm-radio-group accessibility violation issue - Interactive component does not have a programmatically associated name
<img width="1680" alt="ibm-radio-AAT-issue" src="https://user-images.githubusercontent.com/69956330/100838061-cd0fc180-3497-11eb-8df8-f2a9875e211d.png">
